vignettes/sensitivity-analysis.Rmd

```diff
@@ -95,13 +102,14 @@ sensitivites <- saResult$allPKParameterSensitivitiesFor(pkParameterName = "AUC_i
 print(sensitivites)
 ```
 The value `-1` for the sensitivity of "AUC_inf" of the output `Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)` for the parameter `Organism-Kidney-Volume` means that increasing `Organism-Kidney-Volume` by 10% will decrease "AUC_inf" by 10%.
+Note that the list of sensitivities is ordered from largest to smallest with respect to magnitude.
 
 ## Import and export of sensitivity analysis results
 Sensitivity analysis calculated in R can be exported to a *.csv file, which can be loaded in another instance.
 
 ```{r SAExport}
 # Load and run the simulation
-simFilePath <- file.path(getwd(), "..", "tests", "data", "Aciclovir.pkml", fsep = .Platform$file.sep)
+simFilePath <- file.path(getwd(), "..", "tests", "data", "simple.pkml", fsep = .Platform$file.sep)
```

Comment:
This should speed up building of vignettes (https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/392)